agent-dynamic-configuration/AgentDynamicConfiguration.proto

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.agent.dynamic.configuration.v3";
+option csharp_namespace = "SkyWalking.NetworkProtocol";
+option go_package = "skywalking/network/agent/configuration/v3";
+
+import "common/Common.proto";
+
+// Query current service dynamic configuration items from OAP.
```

Comment:
```suggestion
// Fetch the latest dynamic configurations of the service.
```

Reply:
Also, we need more documents to describe what the Command should look like. And how UUID system works.